### PR TITLE
#170122283 Post of redflag(corruption) or intervention

### PR DESCRIPTION
**Pull request by Ally4**
#### What does this PR do?

- `Help user to post redflag(corruption) or intervention`

#### Description of Task to be completed?

- `Create the endpoint of posting redflags(corruptions) or interventions`

#### How should this be manually tested?

- `By post in the post and following the instructions in the readme `

#### Any background context you want to provide?

- `Creation of the router`

#### What are the relevant pivotal tracker stories?

- `https://www.pivotaltracker.com/story/show/170122283`

#### Screenshots (if appropriate)
Screenshot (3073)
Screenshot (3074)

### CODE REVIEW

File: server/Version-2/router/redflags-interventions.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import RedflagsAndInterventions from '../controllers/reportsRedflags-interventions';
+import redflagsInterventionmiddle from '../middleware/redflagsInterventions-middleware';
```
Comment (YvesIraguha) [inline server/Version-2/router/redflags-interventions.js]: @Ally4 can you change the naming here. Remove the `-` between words, and then, continue with camelCase naming conventions.

File: server/Version-2/controllers/reportsRedflags-interventions.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        message: error.details[0].message.replace(/"/g, ''),
+      });
+    }
+    const {
+      email, createdby, title, type, latcoordonate, longcoordonate, comment,
+    } = req.body;
+
+    const images = req.files.images[0].path;
+    const videos = req.files.videos[0].path;
+    const report = await pool.query('INSERT INTO reports (email, createdby, title, type, latcoordonate, longcoordonate, images, videos, comment) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)', [email, createdby, title, type, latcoordonate, longcoordonate, images, videos, comment]);
```
Comment (YvesIraguha) [inline server/Version-2/controllers/reportsRedflags-interventions.js]: @Ally4 can you create a separate file, where you can have these `SQL` queries as constants, and then import it to use it in these files, where you are executing them.

File: server/Version-2/router/redflags-interventions.js (hunk 1 above)
Comment (Ally4) [reply server/Version-2/router/redflags-interventions.js]: ok

File: server/Version-2/controllers/reportsRedflags-interventions.js (hunk 2 above)
Comment (Ally4) [reply server/Version-2/controllers/reportsRedflags-interventions.js]: ok

Comment (Ally4) [reply server/Version-2/controllers/reportsRedflags-interventions.js]: Screenshot (3076)
Screenshot (3077)
Screenshot (3078)
Screenshot (3079)
Screenshot (3080)
Screenshot (3081)
Screenshot (3082)

File: server/Version-2/router/redflags-interventions.js (hunk 1 above)
Comment (Ally4) [reply server/Version-2/router/redflags-interventions.js]: Screenshot (3075)

**Pull request merged**